api/feed/v1/feed.proto

```diff
@@ -57,4 +75,6 @@ message PostsRequest {
 
 message PostsResponse {
     repeated Post posts = 1;
+    repeated AggregatedPost aggregatedPosts = 2;
+    bool is_aggregated = 3;
 }
```

Comment:
check protobuf linter (it runs in `check-codegen` CI job)

Reply:
Now is okay for this check. I needed to commit some files from another services like weshnet weird

Reply:
I think the extra changes are due to a yarn.lock version change probably by some IDE